.github/workflows/admin.yml

```
@@ -5,17 +5,37 @@ on:
       - trunk
   pull_request:
 
+env:
+  PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: 'true'
+  COMPOSER_ROOT_VERSION: 6.6.9999999-dev
+
 jobs:
   unit:
     name: Jest
-    uses: shopware/github-actions/.github/workflows/admin-jest.yml@main
-    with:
-      extensionName: ${{ github.event.repository.name }}
-      uploadCoverage: true
-    secrets:
-      codecovToken: ${{ secrets.CODECOV_TOKEN }}
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup extension
+        uses: shopware/github-actions/setup-extension@main
+        with:
+          extensionName: SwagExtensionStore
+          installAdmin: true
```

This input-ID [changed](https://github.com/shopware/github-actions/blob/main/setup-extension/action.yml#L61) recently:

```suggestion
          install-admin: true
```